### PR TITLE
Added adobe feather editor to media-bundle

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | https://github.com/sulu/sulu/pull/2908, https://github.com/massiveart/husky/pull/733
| License | MIT
| Documentation PR | -

#### What's in this PR?

This PR adds the adobe creative sdk to the media-bundle. A JavaScript-Service provides methods which allow for editing images in the aviary editor.

#### Example Usage

~~~javascript
define([
    'services/sulumedia/image-editor'
], function(
    imageEditor
) {

    'use strict';

    imageEditor.editImage('https://example.org/my-image.jpg').then(function(urlToTheEditedImage) {
        // Do something with the edited image
    });
});

~~~

#### To Do

- [x] Embed Adobe creative sdk
- [x] Create JS-service which provides the possibility to edit images
- [x] Use the JS-service in the media-edit overlay
- [x] Make the Adobe creative API configurable via the Symfony config files
- [x] Translations (EN, DE, NL, FR)

### CODE REVIEW

**Comment by marcelmoosbrugger** [comment]
@danrot The work done in this PR is finished. Let me know when you have merged all the dependent PRs, so I can rebase this one.

**Comment by wachterjohannes** [comment]
@danrot i think you should review this (: i think you have reviewed most of the PR for this feature in recently

**Comment by danrot** [comment]
AFAIC this PR is not completely finished, I have to do that, and it should be reviewed by somebody else afterwards.

**Comment by wachterjohannes** [comment]
ok thats no problem give me a hint when your are finished. we can then talk about that feature before i will review it.

**Comment by marcelmoosbrugger** [comment]
@danrot The PR and the feature is essentially complete. Only small improvements could be added if desired. For example the possibility of adding "stickers" could be removed via tweaking some options of the plugin.

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger we need that feature (:

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js [hunk 1]
```diff
@@ -518,6 +548,24 @@ define([
         },
 
         /**
+         * Sets the new file version for a media by a given url. Shows
+         * a confirmation dialog to the user to confirm the change.
+         *
+         * @param {String} newMediaUrl The url to the new media
+         */
+        setNewVersionByUrl: function(newMediaUrl) {
+            this.sandbox.emit(
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js]: Maybe we should use `this.sandbox.sulu.showConfirmationDialog` instead.

**Comment by danrot** [comment]
@marcelmoosbrugger Ah, cool, I thought the thing with the API key wasn't implemented yet :-)

**Comment by danrot** [comment]
Replaced by #2951

**Pull request closed without merging**